web/src/app/services/classifications/classifications.service.ts

```diff
@@ -8,7 +8,7 @@ import { Classification } from 'app/models/classification';
 import { ClassificationDefinition } from 'app/models/classification-definition';
```

Comment:
General UI issues:
- Remove add button if there is no workbasket selected
- You need to validate the form because if you try to save without entering any field the error message is returning an error but no field is displayed as required or needed
- I would suggest place required fields at the beginning of the form (better readibility)
- I am not sure if classificaiton dropdown is bootstrap's default if not, use it (the same as classification screen)
- I think there is no need for showing non-editable fields like creation date ... since you are creating a task and the can't be modified
- I would suggest to add a yellow badge close to the title, like creation of a workbasket(Creating new workbasket)
- Title doesn't change and there is no field wich makes it change
- All references values seems to be mandatory, please validate the form before saving.

Reply:
* done
* own ticket for validation, needs to be done not only when adding tasks
* done
* done
* done
* done
* done
* see 2